schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -88,53 +92,496 @@ protocol CommonInterpreted {
+        Weight categories as described in ACMG guideline
+        */
+        union {null, AcmgEvidenceStrength} strength;
+        /**
+        Intensity of the strength, together define the weight of each creteria, i.e the 2 in PM2
+        */
+        union {null, int} level;
+        /**
+        Short name for each the criteria as define in the guidline. This is the only required value, but with it all the
+        others can be populated
+        */
+        AcmgCriterion criterion;
+        /**
+        Description of the evidence
+        */
+        union {null, string} description;
+    }
 
+    /**
+    Full record for the ACMG variant clasiffication, including all selectedd evidences and the final classification.
+    */
+    record AcmgVariantClassification{
+        array<AcmgEvidence> acmgEvidences;
+        ClinicalSignificance clinicalSignificance;
+        union{null, string} assesment;
+    }
+
+    /**
+    Evidences as defined in AMP guidelines, they are composed by a evidence type (first column in the evidence table of
+    the guidlines) and a assesment of the evicence, this last one will define the streght of the evidence, supporting
+    the variant to be classified as TierI-IV
+    */
+    record AmpEvidence {
         /**
-        Sample identifier where the variant was observed
+        AMP evidence type according to Guidlines, i.e germline_database_presence
         */
-        string sampleId;
+        AmpEvidenceType type;
+        /**
+        Assesment for AMP evidence, i.e Present in ClinVar
+        */
+        string evidenceAssesment;
+    }
 
+    /**
+    Amp Clinical or Experimental Evidence, the level will define the overal clasification of the variant together with
+    the tiering.
+    */
+    record  AmpClincialOrExperimentalEvidence{
         /**
-        Zygosity. For somatic variants, or variants without zygosity use `na`
+        As denined in AMP guidelines: therapeutic, diagnosis or prognosis
         */
-        Zygosity zygosity;
+        AmpClincialOrExperimentalEvidenceCategory category;
+        /**
+        As denined in AMP guidelines: levelA, levelB, levelC, levelD
+        */
+        AmpClincialOrExperimentalEvidenceLevel level;
+        /**
+        Description of the evidence
+        */
+        union {null, string} description;
+    }
 
+    /**
+    Full Variant classification acording to AMP guideline, including all supporting evidences and the final
+    assesment
+    */
+    record AmpVariantClassification{
         /**
-        Phase set of variants when variants are phased. Two variants with the same value in this field are in phase,
-        if the value is null the variant is unphased. These values are only relevant within the same report.
+        List of AMP evidences
         */
-        union {null, int} phaseSet;
+        array<AmpEvidence> ampEvidences;
+        /**
+        Final Clasification taken in account the evidences
+        */
+        AmpTier ampTier;
```

Comment:
seems to me like a different way of doing things if compared with the ACMG model, at least different in naming convention, can we chat about this?